### PR TITLE
Move core functions from restock.lic into common items.

### PR DESCRIPTION
**Pull request by charleyc-dr**
Split into its own commit so people have time to pick this up before scripts start using it.

### CODE REVIEW

**Comment by jayRyan24** [comment]
The function `count_items_in_container` is actually bugged in that it counts the item it is looking up as one of the elements. 


  ```def count_items_in_container(item, container)
    contents = DRC.bput("rummage /C #{item} IN MY #{container}", '^You rummage .*', 'That would accomplish nothing')
    contents.scan(/#{item}/).size
  end```

```[bitchin-pick]>rummage /C box IN MY satchel
You rummage through a deep brown leather satchel with bronze-capped corners looking for something similar to
  "box" and see a moldy wooden strongbox, a deeply gouged ironwood strongbox and a sturdy mahogany strongbox.
>
[bitchin-pick: contents: You rummage through a deep brown leather satchel with bronze-capped corners looking for
  something similar to "box" and see a moldy wooden strongbox, a deeply gouged ironwood strongbox and a sturdy
  mahogany strongbox.]
[bitchin-pick: 4]
--- Lich: bitchin-pick has exited.```

```
[bitchin-pick]>rummage /C box IN MY satchel
You rummage through a deep brown leather satchel with bronze-capped corners looking for something similar to
  "box" and see a moldy wooden strongbox, a deeply gouged ironwood strongbox and a sturdy mahogany strongbox.
>
[bitchin-pick: 4]
```


    ```count = count_items_in_container('box', 'satchel')
    echo "#{count}"
    return```

**Comment by jayRyan24** [comment]
```
[bitchin-pick]>rummage /C poop IN MY satchel
You rummage through a deep brown leather satchel with bronze-capped corners looking for something similar to
  "poop" but there is nothing in there like that.
>
[bitchin-pick: contents: You rummage through a deep brown leather satchel with bronze-capped corners looking for
  something similar to "poop" but there is nothing in there like that.]
[bitchin-pick: count: 1]
```

**Comment by charleyc-dr** [comment]
Good catch @jayRyan24 !  I fixed the regexp so it won't match the quoted string, and also won't match substrings.  LMK if this works better for you.

File: common-items.lic [hunk 1]
```diff
@@ -91,6 +91,41 @@ module DRCI
...
+              count_items(item)
+            when *match_messages
+              DRC.text2num(Regexp.last_match(1).tr('-', ' '))
+            end
+    waitrt?
+    count
+  end
+
+  def count_items(item)
+    /inside your (.*).|I could not find/ =~ DRC.bput("tap my #{item}", 'inside your (.*).', 'I could not find')
```
Comment (rpherbig) [inline common-items.lic]: Will this count items in your hands or worn?

**Pull request merged**